pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.aquality-automation</groupId>
     <artifactId>aquality-selenium-core</artifactId>
-    <version>2.0.0</version>
+    <version>2.0.1</version>
```

Comment:
Probably 2.0.3
https://mvnrepository.com/artifact/com.github.aquality-automation/aquality-selenium-core we already have 2.0.1 and 2.0.2 published

Reply:
Fixed